tests/fetch_data/hostname

```diff
@@ -1 +1 @@
-myhostname
+myhostname
```

Comment:
Any reason to not have both the newline including file and not (I might not be understanding how tests work).

Reply:
The test spawns a web server that serves the literal contents of the files under `tests/fetch_data`. It would be good to restructure the test to support multiple scenarios like that, but not presently supported.